spec/support/coronavirus_feature_steps.rb

```diff
@@ -356,6 +356,21 @@ def then_the_timeline_entry_is_updated
   expect(@timeline_entry.reload.content).to eq("##Form content")
 end
 
+# Deleting timeline entries
+
+def when_i_delete_a_timeline_entry
+  set_up_github_data(@coronavirus_page)
+
+  page.accept_alert "Are you sure?" do
+    page.find("a[href=\"/coronavirus/landing/timeline_entries/#{@timeline_entry.id}\"]", text: "Delete").click
```

Comment:
This seems rather specific would `click_link("Delete")` work?

Reply:
When the [Reorder page PR](https://github.com/alphagov/collections-publisher/pull/1256) is ready, they'll be more than one Delete link in the Timeline section, so I'm going to leave this in ready for the rebase.